src/miner.cpp

```diff
@@ -39,21 +39,21 @@
 #include <random>
 #include <utility>
 
-struct EvmAddressWithNonce {
-    EvmAddress address;
+struct EvmAddressDataWithNonce {
```

Comment:
Would keep the names rather than types, i.e. `EvmAddressWithNonce`

Reply:
Good catch, agree, will change